### PR TITLE
pulseaudio-modules-droid*: fix install directory

### PR DESCRIPTION
**Pull request by Tofee**
With scarthgap the PulseAudio 17.0 modules directory is now /usr/lib/pulseaudio/modules.

### CODE REVIEW

File: meta-luneos/recipes-multimedia/pulseaudio/pulseaudio-modules-droid-hidl_git.bb [hunk 1]
```diff
@@ -18,14 +18,16 @@ SRC_URI = "git://github.com/droidian/pulseaudio-modules-droid-hidl.git;branch=bo
     file://0001-module-hidl-use-PA_MAJORMINOR-as-PA_MODULE_VERSION-.patch \
 "
 
+EXTRA_OECONF = " --with-module-dir=${libdir}/pulseaudio/modules "
```
Comment (shr-project) [inline meta-luneos/recipes-multimedia/pulseaudio/pulseaudio-modules-droid-hidl_git.bb:21]: no need to add spaces all around just for case someone doesn't know how to use :append :)

Comment (Tofee) [reply meta-luneos/recipes-multimedia/pulseaudio/pulseaudio-modules-droid-hidl_git.bb]: Fixed; no idea who you might be talking about ;)

File: meta-luneos/conf/distro/include/defaulttunes.inc [hunk 2]
```diff
@@ -81,3 +81,5 @@ PACKAGE_EXTRA_ARCHS:append:raspberrypi4-64 = " aarch64-rpi"
 # corei7-64 (Intel Nehalem CPU with 64-bit extensions, MMX, SSE, SSE2, SSE3, SSSE3, SSE4.1, SSE4.2 and POPCNT instruction set support. This tune is recommended for Intel Nehalem and Silvermont (e.g. Bay Trail) CPUs (and beyond)).
 # x86-64-v3 (CPUs with AVX, AVX2, BMI1, BMI2, F16C, FMA, LZCNT, MOVBE, XSAVE. (but not AVX512). See https://www.phoronix.com/news/GCC-11-x86-64-Feature-Levels for details.)
 DEFAULTTUNE:qemux86-64 = "corei7-64"
+
+OLDEST_KERNEL = "3.2.0"
```
Comment (shr-project) [inline meta-luneos/conf/distro/include/defaulttunes.inc:85]: This part doesn't belong here.

Comment (Tofee) [reply meta-luneos/conf/distro/include/defaulttunes.inc:85]: oh sorry ! I must have done a git commit -a by mistake. Let me fix that.

**Review by shr-project — APPROVED** [review]

**Pull request merged**